docs/dev/table/connect.md

```diff
@@ -164,21 +164,8 @@ CREATE TABLE MyUserTable (
   'connector.properties.zookeeper.connect' = 'localhost:2181',
   'connector.properties.bootstrap.servers' = 'localhost:9092',
 
-  -- specify the update-mode for streaming tables
```

Comment:
And above "For streaming queries, an update mode declares how to communicate between a dynamic table and the storage system for continuous queries."
need remove or not?

Reply:
I would like to add a following sentence, what do you think?

The connector might already provide a default update mode (e.g. Kafka connector works in append mode by default).